node/src/actors/chain_manager/superblock.rs

```diff
@@ -17,13 +17,15 @@ pub enum AddSuperBlockVote {
 /// State related to superblocks
 #[derive(Debug, Default)]
 pub struct SuperBlockState {
-    // Set of ARS identities that can currently send superblock votes
-    ars_identities: HashSet<PublicKeyHash>,
+    // Set of ARS identities that will be able to send superblock votes in the next superblock epoch
+    current_ars_identities: Option<HashSet<PublicKeyHash>>,
```

Comment:
why an option?

Reply:
Because we don't know the list of ARS identities when starting the node, we must wait for the first superblock mining call. This was incorrect before.